src/item/views/Item.tsx

```diff
@@ -107,59 +99,19 @@ const Item: FunctionComponent<ItemProps> = ({ history, match }) => {
 	// 	setTime(parseInt((queryParams.time as string) || '0', 10));
 	// };
 
-	const handleTimeLinkClicked = async (timestamp: string) => {
```

Comment:
deze laten we beter staan als we ooit de click timestamp to jump to video locatie terug willen implementeren met de flowplayer

mss enkel in commentaar zetten

Reply:
Staat nu in ItemVideoDescription.tsx ipv hier, stond gewoon duplicate waarschijnlijk door merge conflict.